conf/build.manifest

```diff
@@ -72,6 +72,7 @@ base-packages:
 - truenas-samba
 - nfs4xdr-acl-tools
 - netatalk
+- prometheus-node-exporter
```

Comment:
This needs to go under "additional-packages" instead....
And preferably make a Jira suggestion first and link that one to both the PR and the package...

Like this:

```
- package: prometheus-node-exporter
  comment: requested by community (NAS-number)
```

Reply:
Sounds good, I found `NAS-103563`. Perhaps we can re-open it.